### PR TITLE
Made the search panel expandable.

### PR DESCRIPTION
**Pull request by elliotjz**

### CODE REVIEW

File: mobileApp/js/components/SearchBar/SearchBar.js [hunk 1]
```diff
@@ -46,79 +53,114 @@ class SearchBar extends React.Component {
 			},
 			container: {
 				flex: 1,
-				paddingTop: '8%',
-				width: '90%'
+				display: this.props.shouldShowSearchInput || this.state.animating ? 'flex' : 'none',
```
Comment (stan-sack) [inline mobileApp/js/components/SearchBar/SearchBar.js:56]: Great use of || and ternary operator

**Comment by stan-sack** [comment]
the code you have is good. I have a few stylistic changes that i'd like to see.

1. Can you please make the contracted icon round rather than square. also I think it would look a lot better if it was centred.

2. I reckon you should keep the search icon in the expanded search bar and make it trigger a search. its not totally clear that the search is triggered by pressing enter.

3. When the x button is pressed it is highlighted black but its super quick and looks like a bug (even though its just the default behaviour of the TouchableOpacity). I reckon you should disable this.

**Comment by stan-sack** [comment]
Also can you please put some padding to the left of the placeholder text "Search" so it's not up against the side of the input container?

**Comment by elliotjz** [comment]
I couldn't fix the padding issue. I'm a bit lost on it. It looks like the padding is there in the code, and actually fixes itself once you start typing.

**Comment by stan-sack** [comment]
Great stuff. I've merged into master

**Pull request merged**